service/resource/vaultcrt/resource.go

```diff
@@ -1,55 +1,75 @@
 package vaultcrt
 
 import (
+	"time"
+
+	"github.com/giantswarm/certificatetpr"
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger"
 	"github.com/giantswarm/operatorkit/framework"
 	"github.com/giantswarm/vaultcrt"
 	"github.com/giantswarm/vaultrole"
 	"k8s.io/client-go/kubernetes"
 	apiv1 "k8s.io/client-go/pkg/api/v1"
+
+	"github.com/giantswarm/cert-operator/service/key"
 )
 
 const (
-	Name = "vaultcrt"
 	// AllowSubDomains defines whether to allow the generated root CA of the PKI
 	// backend to allow sub domains as common names.
 	AllowSubDomains = true
+	Name            = "vaultcrt"
+	// UpdateTimestampAnnotation is the annotation key used to track the last
+	// update timestamp of certificates contained in the Kubernetes secrets.
+	UpdateTimestampAnnotation = "giantswarm.io/update-timestamp"
+	// UpdateTimestampLayout is the time layout used to format and parse the
+	// update timestamps tracked in the annotations of the Kubernetes secrets.
+	UpdateTimestampLayout = "2006-01-02T15:04:05.000000Z"
```

Comment:
is `Layout` the right term here? I was confused here. I think the better term is `Format`

Reply:
This is the golang naming. 

- https://golang.org/pkg/time/#Parse
- https://golang.org/pkg/time/#Time.Format

Reply:
TIL. Thanks! <3